index.html

```diff
@@ -208,11 +210,11 @@
         .sizeByClinicalAttribute("PSA", "Result")
         .sizeByClinicalAttribute("Phos", "Result")
         .orderAllTooltipTables(["Source","Location"])
-        .enableTrackTooltips(true)
-        .enableZoom(enableAdvanced) //zoom functionality avaiable only in the advance view
-        .advancedView(enableAdvanced)
-        .enableTrimmedTimeline(true)
-        .enableVerticalLine(false)
+        .getOrSetBool("enableTrackTooltips", true)
+        .getOrSetBool("enableZoom", enableAdvanced) 
+        .getOrSetBool("advancedView", enableAdvanced)
+        .getOrSetBool("enableTrimmedTimeline", true)
+        .getOrSetBool("enableVerticalLine", false)
```

Comment:
Actually meant for the `getOrSetBool` to be private, then the other functions call `getOrSetBool. That way you can keep the same public functions.